contracts/strategies/MobiusStrategy.sol

```diff
@@ -147,7 +168,12 @@ contract MobiusStrategy is Ownable, IStrategy {
         minter = _minter;
         mobi = _mobi;
         celo = _celo;
-        lpToken = IERC20(pool.getLpToken());
+        inboundTokenIndex = _inboundTokenIndex;
+        if (address(_gauge) != address(0)) {
+            lpToken = IERC20(_pool.getLpToken());
+        } else {
+            lpToken = _lpToken;
```

Comment:
we should probably validate and enforce that `_lpToken` is not zero in the case there's no gauge set.